src/indicators/flooding/glofas/glofas.py

```diff
@@ -31,7 +33,7 @@ def __init__(
         cds_name: str,
         dataset: List[str],
         dataset_variable_name: str,
-        system_version_minor: int = None,
+        system_version_minor: Dict[int, int],
```

Comment:
Should have picked this up in a past review, but what is ```system_version_minor```?

Reply:
Good question! So when you make the query to CDS you have to specify the model version, which has the format {major}.{minor}. The major version is 3 for the new model, 2 for the old one. Minor is almost always 1, except for the old mode reforecast, where it is 2  -- hence the need to turn it into a dictionary, where the key is the major version number. 

But anyway you're right that this point needs more clarification, so I've added a docstring. In general this class could use a few more docstrings...!